tasks/configuration.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: template isc-dhcp-server dhcpd.conf
+  template:
+    src: dhcpd.conf.j2
+    dest: /etc/dhcp/dhcpd.conf
```

Comment:
Is that always on the same place or is it better to put this path into a variable?

Reply:
So far it's been the same on Ubuntu, Debian, CentOS and RHEL.